bears/markdown/MarkdownBear.py

```diff
@@ -132,21 +132,21 @@ def create_arguments(filename, file, config_file,
             'ruleSpaces': horizontal_rule_spaces,       # Bool
             'ruleRepetition': horizontal_rule_repeat,   # int
         }
-        remark_configs_plugins = {}
+        remark_lint_configs = {}
 
         if max_line_length:
-            remark_configs_plugins['maximumLineLength'] = max_line_length
+            remark_lint_configs['maximumLineLength'] = max_line_length
 
         config_json = json.dumps(remark_configs_settings)
```

Comment:
Maybe we should also rename this to `remark_configs`, so it's easier to distinct from `remark_lint_configs`?

Up to you :+1: